packages/cursorless-engine/src/testCaseRecorder/TestCaseRecorder.ts

```diff
@@ -275,14 +277,14 @@ export class TestCaseRecorder {
       this.spyIde = new SpyIDE(this.originalIde);
       injectIde(this.spyIde!);
 
-      const spokenForm = generateSpokenForm(command);
+      const spokenForm = this.spokenFormGenerator.processCommand(command);
 
       this.testCase = new TestCase(
         {
           ...command,
           spokenForm:
             spokenForm.type === "success"
-              ? spokenForm.value
+              ? spokenForm.spokenForms[0]
```

Comment:
May be add a comment about why we just pick the first spoken form?

Reply:
Yeah I'm not totally in love with all the `[0]`s here, but I don't think it's unreasonable. I'll add a comment and maybe try to unify them